build/barys

```diff
@@ -480,6 +482,30 @@ for pair in $ADDITIONAL_VARIABLES; do
     echo "$variable=\"$value\"" >> conf/local.conf
 done
 
+DEVICE_LAYER_CONF=$(ls ${SCRIPTPATH}/../../layers/meta-balena-*/conf/layer.conf)
```

Comment:
nitpick: Maybe using inverted logic makes this cleaner, that is, defaulting to `USE_OLD_SYNTAX=no` and only switching it when the release is not already supported.